packages/terra-consumer-nav/src/Layout.scss

```diff
@@ -24,9 +24,22 @@
     top: 0;
     width: calc(100vw - #{$terra-consumer--nav-container-width});
     @media screen and (max-width: $terra-medium-breakpoint) {
-      left: 0;
+      left: 20;
       max-width: 100vw;
       width: 100vw;
     }
   }
+  .footer {
+    width: 100%;
+    bottom: 15px;
+    margin:0;
+    padding:0;
+    position:sticky;
```

Comment:
Based on UX review, the help button does not require a sticky position. The help button has fixed position for desktop and the position is relative to page content in mobile/tablet view.